src/Stratis.Bitcoin.Features.BlockStore/BlockRepository.cs

```diff
@@ -18,6 +18,20 @@ namespace Stratis.Bitcoin.Features.BlockStore
     /// </summary>
     public interface IBlockRepository : IBlockStore
     {
+        /// <summary> The dbreeze database engine.</summary>
+        DBreezeEngine DBreeze { get; }
+
+        /// <summary>
+        /// Delete blocks and their transactions.
+        /// <para>
+        /// It should be noted that this does not delete the entries from disk (only the references are removed) and
+        /// as such the file size remains the same.
```

Comment:
Add a TODO to fix this since it's a dbreeze problem, not repository's

Reply:
Fixed